src/elements/Header/HeaderSubheader.js

```diff
@@ -1,16 +1,22 @@
-import React, { PropTypes } from 'react'
 import cx from 'classnames'
-import { getElementType, getUnhandledProps, META } from '../../lib'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getElementType,
+  getUnhandledProps,
+  META,
+} from '../../lib'
 
 function HeaderSubheader(props) {
-  const { children, className } = props
+  const { children, className, content } = props
   const classes = cx('sub header', className)
   const rest = getUnhandledProps(HeaderSubheader, props)
   const ElementType = getElementType(HeaderSubheader, props)
 
   return (
     <ElementType className={classes} {...rest}>
-      {children}
+      {children || content}
```

Comment:
In this case, having a single return is OK because either children or content will be rendered.  But, never will children _and_ shorthand be rendered.  👍